hebiros/CMakeLists.txt

```diff
@@ -242,10 +256,9 @@ target_link_libraries(hebiros_node ${catkin_LIBRARIES} ${PROJECT_SOURCE_DIR}/lib
 #############
 
 ## Add gtest based cpp test target and link libraries
-# catkin_add_gtest(${PROJECT_NAME}-test test/test_hebiros.cpp)
-# if(TARGET ${PROJECT_NAME}-test)
-#   target_link_libraries(${PROJECT_NAME}-test ${PROJECT_NAME})
-# endif()
+find_package(rostest REQUIRED)
```

Comment:
Is this where `gtest` was getting included from?

Reply:
Nope -- that was on a completely different repository.  This is for the unit tests we have set up here.